src/figuresClasses.ts

```diff
@@ -1,19 +1,105 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 export interface Figure {
-
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  color: Color;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(color: Color, a: number, b: number, c: number) {
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
```

Comment:
```suggestion

  constructor(public color: Color, public a: number, public b: number, public c: number) {
```
it can be shorter. and work in constructor with public properties (without this)